pynicotine/pynicotine.py

```diff
@@ -297,6 +298,7 @@ def quit(self, signal_type=None, _frame=None):
 
         config.write_configuration()
 
+        log.add_debug("Saved configuration: %(file)s", {"file": config.filename})
```

Comment:
In my opinion, this would make more sense to include in write_configuration.

Reply:
`log` is not imported in `config`